src/manage_nsfs/health.js

```diff
@@ -480,6 +482,45 @@ class NSFSHealth {
         }
     }
 
+    /**
+     * _get_blocked_hosts_status checks if there are any hosts blocked for updating the config directory
+     * 1. when host's config_dir_version older than 5.18.0 and system's config_dit_version is 5.18.0 and higher it means it's not blocked 
```

Comment:
should we document it as 5.18.0 hard-coded? (we will not use those functions in future upgrades?)

Reply:
1 & 2 are specific cases of when we don't have config dir version of the system/of a specific host, that should be added on 5.18.0 upgrade
I added bullet 3 of the general case